source/JarResolverLib/src/Google.JarResolver/PlayServicesSupport.cs

```diff
@@ -502,12 +502,11 @@ public void ClearDependencies()
 
                         // Extract the version from the filename.
                         // dep.Artifact is the name of the package (prefix)
-                        // The regular expression extracts the version number from the filename
-                        // handling filenames like foo-1.2.3-alpha.
-                        match = System.Text.RegularExpressions.Regex.Match(
-                            filenameWithoutExtension.Substring(
-                                dep.Artifact.Length + 1), "^([0-9.]+)");
-                        if (match.Success)
+                        string artifactVersion = ExtractVersionFromFileName(
```

Comment:
Not sure why you need this.  The previous code attempted to extract the version from the filename and if the match succeeds then jumps into the block below and pulls the version from match.Groups[1].Value.  Did you see a problem in this code path or was this just a clean up?